images/virtualization-artifact/pkg/controller/supplements/ensure.go

```diff
@@ -60,7 +64,7 @@ func EnsureForPod(ctx context.Context, client client.Client, supGen *Generator,
 	}
 
 	// Create Secret with auth config to use DVCR as destination.
-	if ShouldCopyDVCRAuthSecret(dvcrSettings, supGen) {
+	if forceCopyDVCRAuthSecret || ShouldCopyDVCRAuthSecret(dvcrSettings, supGen) {
```

Comment:
forceCopyDVCRAuthSecret seems like another quick hack. This logic should be a part of ShouldCopyDVCRAuthSecret method.

Reply:
Fixed.